roles/openshift_certificate_expiry/test/test_load_and_handle_cert.py

```diff
@@ -0,0 +1,68 @@
+'''
+ Unit tests for the load_and_handle_cert method
+'''
+import datetime
+import os
+import sys
+
+import pytest
+
+module_path = os.path.normpath(os.path.join(__file__, os.pardir, os.pardir, 'library'))
```

Comment:
Why did you choose `normpath` instead of `realpath` that we use elsewhere?

---

Side note, if we run pylint through this file, it will complain about the invalid name `module_path`, about the import order and import erorr :-)  #3511